cfg/AvtVimbaCamera.cfg

```diff
@@ -160,6 +160,7 @@ exposure_mode_enum = gen.enum([gen.const("Timed", str_t, "Timed", "[Default] Cam
 # ACQUISITION
 gen.add("acquisition_mode",       str_t,    SensorLevels.RECONFIGURE_STOP,    "Camera acquisition mode", "Continuous", edit_method = acquisition_mode_enum)
 gen.add("acquisition_rate",       double_t, SensorLevels.RECONFIGURE_RUNNING, "Sets the expected triggering rate in externally triggered mode.", 10, 1, 35)
+gen.add("acquisition_rate_enable",       bool_t, SensorLevels.RECONFIGURE_RUNNING, "Allows to fix acquisition rate, if available", False)
```

Comment:
Can you adjust the whitespace formatting here so that bool_t aligns vertically with all the other field types?